harness/resizableArrayBufferUtils.js

```diff
@@ -18,14 +18,10 @@ defines:
 features: [BigInt]
 ---*/
 
-class MyUint8Array extends Uint8Array {
-}
-
-class MyFloat32Array extends Float32Array {
-}
-
-class MyBigInt64Array extends BigInt64Array {
-}
+// Using new Function()(); instead of just 'class x extends Y' as to not bomb out when `class` isn't supported
+const MyUint8Array = new Function('return class MyUint8Array extends Uint8Array {}')();
+const MyFloat32Array = new Function('return class MyFloat32Array extends Float32Array {}')();
+const MyBigInt64Array = new Function('return class MyBigInt64Array extends BigInt64Array {}')();
```

Comment:
is it ok that these exceptions aren't caught?

Reply:
Sigh, no, they are not. I'm sorry, juggling too many things at once and trying to rush it through, shouldn't have done that...

Will send an additional PR tomorrow... Sorry again for keeping you all occupied...